charts/testrunnerdashboard/templates/_labels.tpl

```diff
@@ -0,0 +1,14 @@
+{{- /*
+Create labels to use for resources in this chart
+
+See
+https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
+*/ -}}
+{{- define "testrunnerdashboard.labels" -}}
+helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+app.kubernetes.io/name: {{ .Chart.Name }}
+app.kubernetes.io/instance: {{ .Values.name | quote }}
+app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+app.kubernetes.io/component: testrunnerdashboard
+app.kubernetes.io/part-of: terra
```

Comment:
there's a chance that this specifically might change. We're not planning on deploying this to a Terra environment--we'll be deploying it to a new cluster, dsp-eng-tools. `part-of: tools` might make more sense here